### PR TITLE
Fix(rr): Non-static method RRHelper::extractZIP() should not be calle…

### PR DESCRIPTION
**Pull request by ovr**
…d statically

### CODE REVIEW

**Comment by wolfy-j** [comment]
You have changed the signature of incorrect method.

File: src/bin/rr [hunk 1]
```diff
@@ -132,7 +132,7 @@ class RRHelper
      * @param string $target
      * @throws Exception
      */
-    public static function extractBinary(string $archive, string $target)
+    protected static function extractZIP(string $archive, string $target)
```
Comment (wolfy-j) [inline src/bin/rr]: Incorrect method.

Comment (ovr) [reply src/bin/rr]: oops!

Comment (wolfy-j) [reply src/bin/rr]: if would like to make it static, can you please do the same for TAR method

**Comment by wolfy-j** [comment]
Thank you!

**Pull request merged**